x-pack/auditbeat/module/system/user/user_test.go

```diff
@@ -14,6 +14,8 @@ import (
 )
 
 func TestData(t *testing.T) {
+	t.Skip("test is failing in CI")
```

Comment:
May be useful to link to the initial report #9679 in this message. The issue contains good information on when this happens. Or even directly say that in the skip message.

Reply:
I agree, it would have been nice. This time, `git blame` will find it for those that are interested.